src/contractTest/java/uk/gov/hmcts/reform/managecase/consumer/CCDConsumerTest.java

```diff
@@ -0,0 +1,201 @@
+package uk.gov.hmcts.reform.managecase.consumer;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.dsl.PactDslJsonBody;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactConsumerTestExt;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import com.google.common.collect.ImmutableMap;
+import io.restassured.RestAssured;
+import io.restassured.path.json.JsonPath;
+import org.apache.http.entity.ContentType;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(PactConsumerTestExt.class)
+public class CCDConsumerTest {
```

Comment:
I think it would be better to have a dedicated class for each distinct API operation. This allows us to manage the pact for each operation in isolation from others. we can also apply class inheritance to re-use common logic, constants, etc.